app/src/main/java/fr/free/nrw/commons/customselector/ui/adapter/ImageAdapter.kt

```diff
@@ -364,6 +367,47 @@ class ImageAdapter(
         notifyDataSetChanged()
     }
 
+    fun clearSelectedImages(){
```

Comment:
Would you mind adding doc to this method and the next?

Reply:
sure